protocol/x/perpetuals/keeper/perpetual_test.go

```diff
@@ -1383,6 +1383,7 @@ func TestGetSettlement_Success(t *testing.T) {
 				newFundingIndex,
 			)
 
+			bigNetSettlement := bigNetSettlementPpm.Div(bigNetSettlementPpm, lib.BigIntOneMillion())
```

Comment:
Instead of dividing one million here, directly check the ret value (update test case accordingly). Since we should be testing the return value of  `pc.PerpetualsKeeper.GetSettlement`, not ` `pc.PerpetualsKeeper.GetSettlement() / 1_000_000`.